[Harbor] add storageClass config options for PVCs (#2149)
aws/eks-anywhere: docs/content/en/docs/reference/packagespec/harbor/v2.5.0.md

| `persistence.resourcePolicy`                                         | Setting it to `keep` to avoid removing PVCs during a helm delete operation. Leaving it empty will delete PVCs after the chart deleted. Does not affect PVCs created for internal database and redis components.                                                                                                                                                                                                                                                   | `keep`                                |
| `persistence.persistentVolumeClaim.registry.size`                    | The size of the volume                                                                                                                                                                                                                                                                                                                                                                                                                                            | `5Gi`                                 |

> | `persistence.persistentVolumeClaim.registry.storageClass`            | Specify the `storageClass` used to provision the volume. Or the default StorageClass will be used (the default). Set it to `-` to disable dynamic provisioning                                                                                                                                                                                                                                                                                                    |                                       |

Reviewer: Did you mean to actually fill in the defaults here (and several spots below)?

> | `persistence.persistentVolumeClaim.registry.storageClass`            | Specify the `storageClass` used to provision the volume. Or the default StorageClass will be used (the default). Set it to `-` to disable dynamic provisioning                                                                                                                                                                                                                                                                                                    |                                       |

Reviewer: ```suggestion
| `persistence.persistentVolumeClaim.registry.storageClass`            | Specify the `storageClass` used to provision the volume. Defaults to X. Dynamic provision can be disabled by setting it to `-`.                                                                                                                                                                                                |                                       |
```
